packages/itwinui-react/src/core/Tabs/Tabs.tsx

```diff
@@ -177,46 +86,147 @@ export const Tabs = (props: TabsProps) => {
   }
 
   const {
-    labels,
-    activeIndex,
-    onTabSelected,
-    focusActivationMode = 'auto',
-    type = 'default',
-    color = 'blue',
+    className,
+    children,
     orientation = 'horizontal',
-    tabsClassName,
-    contentClassName,
-    wrapperClassName,
+    type = 'default',
+
+    ...rest
+  } = props;
+
+  const [activeValue, setActiveValue] = React.useState<string | undefined>();
+  const [stripeProperties, setStripeProperties] = React.useState({});
+
+  return (
+    <Box
+      className={cx('iui-tabs-wrapper', `iui-${orientation}`, className)}
+      style={stripeProperties}
+      ref={ref}
+      {...rest}
+    >
+      <TabsContext.Provider
+        value={{
+          orientation,
+          type,
+          activeValue,
+          setActiveValue,
+          overflowOptions,
+          setStripeProperties,
+        }}
+      >
+        {children}
+      </TabsContext.Provider>
+    </Box>
+  );
+}) as PolymorphicForwardRefComponent<'div', TabsWrapperOwnProps>;
+TabsWrapper.displayName = 'Tabs.Wrapper';
+
+// ----------------------------------------------------------------------------
+// Tabs.TabList component
+
+type TabListOwnProps = {
+  /**
+   * Color of the bar on the active tab.
+   * @default 'blue'
+   */
+  color?: 'blue' | 'green';
+  /**
+   * Control whether focusing tabs (using arrow keys) should automatically select them.
+   * Use 'manual' if tab panel content is not preloaded.
+   * @default 'auto'
+   */
+  focusActivationMode?: 'auto' | 'manual';
+  /**
+   * Tab items.
+   */
+  children: React.ReactNode[] | React.ReactNode;
+  activeValue?: string;
+  setActiveValue?: (value: string) => void;
+};
+
+const TabList = React.forwardRef((props, ref) => {
+  const {
+    className,
     children,
+    color = 'blue',
+    focusActivationMode = 'auto',
+    activeValue: activeValueProp,
+    setActiveValue: setActiveValueProp,
     ...rest
   } = props;
 
-  const isClient = useIsClient();
+  const {
+    orientation,
+    type,
+    activeValue: activeValueContext,
+    setActiveValue: setActiveValueContext,
+    overflowOptions,
+    setStripeProperties,
+  } = useSafeContext(TabsContext);
+
+  const items = React.useMemo(
+    () => (Array.isArray(children) ? children : [children]),
+    [children],
+  );
 
-  const tablistRef = React.useRef<HTMLUListElement>(null);
+  const activeValue = activeValueProp || activeValueContext;
```

Comment:
Why is `activeValue` needed from both the context and the prop to `TabsList`?

Reply:
Not needed. This was changed.